iOS6-HW12-Alexander.Strelkov/circularProgressBarView.swift

```diff
@@ -21,7 +20,7 @@ class CircularProgressBarView: UIView, CAAnimationDelegate {
             super.init(coder: coder)
         }
 
-func createWorkCircularPath() {
+func createCircularPath() {
         // created circularPath for circleLayer and progressLayer
     let circularPath = UIBezierPath(arcCenter: CGPoint(x: frame.size.width / 2.0, y: frame.size.height / 2.0), radius: 80, startAngle: -Double.pi / 2, endAngle: 3 * Double.pi / 2, clockwise: true)
```

Comment:
Для лучшей читаемости можно параметры расположить друг под другом
```suggestion
    let circularPath = UIBezierPath(arcCenter: CGPoint(x: frame.size.width / 2.0,y: frame.size.height / 2.0), 
                                                          radius: 80, 
                                                          startAngle: -Double.pi / 2, 
                                                          endAngle: 3 * Double.pi / 2, 
                                                          clockwise: true)
```